modules/installation-restricted-network-samples.adoc

```diff
@@ -34,7 +34,7 @@ endif::[]
 
 Setting the `samplesRegistry` field in the Sample Operator configuration file to link:https://registry.redhat.io[registry.redhat.io] is redundant because it is already directed to link:https://registry.redhat.io[registry.redhat.io] for everything but Jenkins images and imagestreams. It also breaks the installation payload for Jenkins imagestreams.
 
-The Samples Operator prevents the use of the following registries for the Jenkins imagestreams:
+The Cluster Samples Operator prevents the use of the following registries for the Jenkins imagestreams:
```

Comment:
This part need removed too. @gabemontero Could you help confirm it?

Reply:
No, samples operator still forces the jenkins imagestreams to come from the payload.

Reply:
if you want to change this to say we don't allow update of jenkins images in general, and no longer list those registries, I'm fine with that as well.

Reply:
Commented the list out, that way if it's needed down the line we can uncomment.